docs/tutorials/podman_tutorial.md

```diff
@@ -108,7 +107,7 @@ system. When transferring the checkpoint, it is possible to specify an output-fi
 
 On the source system:
 ```console
-sudo podman container checkpoint <container_id> -e /tmp/checkpoint.tar.gz
+sudo podman container checkpoint -l -e /tmp/checkpoint.tar.gz
```

Comment:
I'm not sure if you've replaced all of the `<container_id>` tags with lowercase `-l` or not, but if you have, please revert some of them, especially near the top of the document.  It's important to show that IMO.